### PR TITLE
Update options for Split and SectionHeading components for FMO homepage

### PR DESCRIPTION
**Pull request by stevejalim**
* 3/4 split size now supported in standard and reversed
* Section header made to be h2-h4 with alignment option

See the TODOs for areas where we need some media queries or changes so that the left- and right-aligned section  headers don't break grid

(needs latest DB dump loading - will Slack that to you @craigcook )

### CODE REVIEW

**Comment by craigcook** [comment]
I've tidied up the styling and updated the section heading alignment options. Hopefully I did that right...

File: birdbox/microsite/migrations/0068_alter_generalpurposepage_content_and_more.py [hunk 1]
```diff
@@ -144,7 +144,7 @@ class Migration(migrations.Migration):
                                     "alignment",
                                     wagtail.blocks.ChoiceBlock(
                                         blank=True,
-                                        choices=[("", "Center"), ("mzp-c-section-heading-left", "Left"), ("mzp-c-section-heading-right", "Right")],
+                                        choices=[("", "Default"), ("t-align-center", "Center"),],
```
Comment (stevejalim) [inline birdbox/microsite/migrations/0068_alter_generalpurposepage_content_and_more.py:147]: Editing migrations for streamblocks is ok, but it's often simpler to make a new one with `just makemigrations`

**Comment by craigcook** [comment]
I think I successfully added size options (and renamed the level option for clarity). I made a new migration and it seems to work? Let me know if I've broken something. And since these options live in the DB it might take some finagling for you to get it... I can try to send you a dump of my local DB but you'll have to walk me through making it. Or failing that I presume you could remove any section headings in your local install first, pull the commit, run the migration, then add them anew?

**Review by stevejalim — COMMENTED** [review]
r+ thanks @craigcook

**Pull request merged**